Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -67,6 +69,7 @@ import SplitDirection from "../SplitDirection.js";
  * @param {ShadowMode} [options.shadows=ShadowMode.ENABLED] Determines whether the model casts or receives shadows from light sources.
  * @param {Boolean} [options.showCreditsOnScreen=false] Whether to display the credits of this model on screen.
  * @param {SplitDirection} [options.splitDirection=SplitDirection.NONE] The {@link SplitDirection} split to apply to this model.
+ * @param {Boolean} [options.projectTo2D=false] Whether to accurately project the model's positions in 2D mode. This disables minimumPixelSize and prevents future modification to its model matrix. This cannot be set after the model has loaded.
```

Comment:
This will be true when I do the PR for individual models, but it's not true now. Until then, though, I'll add a note that the model won't show up without this flag enabled.